specs/2025-02-24.yaml

```diff
@@ -2,140 +2,79 @@ openapi: 3.0.0
+        The Get Organization API should be used to retrieve detailed information about an organization.
+      operationId: searchOrganizations
       tags:
-        - Nonprofits
+        - Organizations
       security:
         - bearerAuth: []
       parameters:
         - name: ein
-          in: path
+          in: query
           description: |-
-            The unique federal employer identification number (EIN) of the nonprofit.
-            This value should be exactly 9 digits and should not contain any special characters such as dashes.
+            The Employer Identification Number (EIN) or TaxID for the nonprofit entity.
           schema:
             type: string
-          required: true
-          example: "530196605"
-      responses:
-        "200":
-          description: "OK"
-          headers:
-            X-Request-Id:
-              $ref: "#/components/headers/X-Request-Id"
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/Nonprofit"
-              examples:
-                RedCross:
-                  $ref: "#/components/examples/NonprofitRedCross"
-        "400":
-          $ref: "#/components/responses/BadRequestError"
-        "401":
-          $ref: "#/components/responses/AuthenticationError"
-        "403":
-          $ref: "#/components/responses/ForbiddenError"
-        "404":
-          $ref: "#/components/responses/NotFoundError"
-        "500":
-          $ref: "#/components/responses/InternalServerError"
-  /v1/nonprofits:
-    post:
-      summary: Create nonprofit
-      description: |-
-        Create a nonprofit organization.
-
-        This is useful for integration partners to use after a nonprofit consents to use the Chariot payment option on their donation forms.
-
-        <Tip>
-        If a nonprofit does not already exist for the EIN, this will return a `201 Created` status.
-        If a nonprofit already exists for the given EIN on the system, this will return a `200 OK` status.
-        </Tip>
-
-        <Warning>
-        Handling errors:
-        - If the nonprofit does not exist within Chariot's database, a `404 Not Found` status is returned.
-        - If the nonprofit exists but does not pass Chariot's compliance checks, a `412 Precondition Failed` status is returned with a reason.
-        </Warning>
-      operationId: create-nonprofit
-      tags:
-        - Nonprofits
-      security:
-        - bearerAuth: []
-      requestBody:
-        $ref: "#/components/requestBodies/CreateNonprofitRequest"
+        - name: name
+          in: query
+          description: |-
+            The name of the organization.
+          schema:
+            type: string
+        - name: limit
+          in: query
+          description: Limit the size of the list that is returned. The default (and maximum) is 100 objects.
+          required: false
+          schema:
+            type: integer
+            format: int32
```

Comment:
are we not going to support a next page token here?

Reply:
on search? no i don't think we need or want to support pagination, this isnt' a List endpoint

Reply:
search again with a more refined name?